pkg/query-service/app/querier/v2/querier.go

```diff
@@ -171,6 +173,7 @@ func (q *querier) runBuilderQueries(ctx context.Context, params *v3.QueryRangePa
 
 	wg.Wait()
 	close(ch)
+	zap.L().Info("time taken to run builder queries", zap.Duration("duration", time.Since(now)))
```

Comment:
we already have duration for elapsed time, rename it to sometime else ? like querier duration it will be easier to distinguish while filtering from the ui ?